src/coreComponents/physicsSolvers/solidMechanics/kernels/ExplicitSmallStrain_impl.hpp

```diff
@@ -129,11 +130,12 @@ void ExplicitSmallStrain< SUBREGION_TYPE, CONSTITUTIVE_TYPE, FE_TYPE >::quadratu
   real64 const detJ = FE_TYPE::inverseJacobianTransformation( q, stack.xLocal, invJ );
 
   real64 strain[6] = {0};
+  real64 timeIncrement;
```

Comment:
I would suggest to initialize this variable to avoid issues later

Reply:
You're right.